Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -149,10 +150,15 @@ YUI.add('ez-platformuiapp', function (Y) {
          *
          * @method _fireAppReadyEvent
          * @private
+         * @param {Object} event facade
          */
-        _fireAppReadyEvent: function () {
+        _fireAppReadyEvent: function (event) {
             var evt = new CustomEvent('ez:yui-app:ready');
 
+            if (!event.newVal) {
```

Comment:
is that really needed ? as far as I can tell, `initialized` is never set to something else than `true`